docs/reference/default-theme-config.md

```diff
@@ -184,8 +184,10 @@ export type SidebarItem = {
 - Default: `true`
 - Can be overridden per page via [frontmatter](./frontmatter-config#aside)
 
+This is typically used to show the "On this page" section, containing links to major headings within the current page.
+
 Setting this value to `false` prevents rendering of aside container.\
-Setting this value to `true` renders the aside to the right.\
+Setting this value to `true` (the default) renders the aside to the right.\
```

Comment:
no need to specify again,`` - Default: `true` ``  is already written above.

Reply:
Yes, that's true. I've removed it.